firebase.json

```
@@ -486,6 +486,7 @@
       { "source": "/go/mixin-modifier", "destination": "https://docs.google.com/document/d/1la2S1uJrNAawc1yja8hrSxFZJ0cFHnYopmaBYDLwMjg/edit?usp=sharing&resourcekey=0-br1LWMlYC63Q30VGiPtAHg", "type": 301 },
       { "source": "/go/mouse-tracker-no-longer-attaches-annotations", "destination": "https://docs.google.com/document/d/1YuOcepqZjNknAMz1si2m-jAiLT4MP28XEzBcKLHnNiA/edit", "type": 301 },
       { "source": "/go/move-mouse-tracker-to-rendering", "destination": "https://docs.google.com/document/d/1dYNOTZ4kdq6ndNHopjUlrGlGBw3W-3jMbnoNJouIYtE/edit", "type": 301 },
+      { "source": "/go/move-useful-flutter-types", "destination": "https://docs.google.com/document/d/1qQokfeExIx3E7X_2ogFfZ4U5viBQflwVSKtlazPjgOY/edit?usp=sharing&resourcekey=0-XFum7OfSG2Nkvvq8w5QxdA", "type": 301 },
```

Not necessary, but consider a redirect name that contextualizes what types you are discussing.

Perhaps something like:

```suggestion
      { "source": "/go/move-flutter-agnostic-types", "destination": "https://docs.google.com/document/d/1qQokfeExIx3E7X_2ogFfZ4U5viBQflwVSKtlazPjgOY/edit?usp=sharing&resourcekey=0-XFum7OfSG2Nkvvq8w5QxdA", "type": 301 },
```

---

....oh, you clever person. sure

I'll have to rename my doc and everything! *sigh* 😜